src/Rules/CheckClassNamespaceFollowPsr4Rule.php

```diff
@@ -87,7 +88,7 @@ public function processNode(Node $node, Scope $scope): array
         $namespacePart = substr($namespaceBeforeClass, 0, -1);
         $errorMessage = sprintf(self::ERROR_MESSAGE, $namespacePart);
 
-        return [$errorMessage];
+        return [RuleErrorBuilder::message($errorMessage)->build()];
```

Comment:
identifier name should be set?

```
->identifier('check.classnamespacepsr4')
```

Reply:
fixed https://github.com/symplify/phpstan-rules/pull/144/commits/c8b251300eed3a078648f2a8220e841a7516ac8e